lib/common/http1client.c

```diff
@@ -660,30 +660,9 @@ static h2o_iovec_t build_request(struct st_h2o_http1client_t *client, h2o_iovec_
 #undef APPEND_STRLIT
 }
 
-static void on_connection_ready(struct st_h2o_http1client_t *client)
+static void start_request(struct st_h2o_http1client_t *client, h2o_iovec_t method, const h2o_url_t *url,
+                          const h2o_header_t *headers, size_t num_headers, h2o_iovec_t body, h2o_httpclient_properties_t props)
```

Comment:
I think `props` should be a pointer?

Reply:
I see. Fixed 4487f78d5fea2cf5733c30afbbf1b7950fe31b47